Assignment.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import './App.css';
+import { v4 as uuidv4 } from 'uuid';
+uuidv4();
+
+const Assignment = () => {
+  const [input, setInput] = useState("")
+  const [submittedInput, setSubmittedInput] = useState([])
+
+  const handleChange = (e) => {
+    setInput(e.target.value) // Updates state every key stroke
+  }
+  
+  const handleClick = () => {
+    setSubmittedInput(input) // updates our list of submitted input with the typed word from the state "input"
+  }
+
+  const inputList = submittedInput.map((input) => {
+    <p key={uuidv4}>{input}</p>

```

Comment:
YES! kick ass! You got npm imports going and we able to use them to generate unique ids. However, I think you need to call `uuidv4()` in the key. I might be wrong. Also you don't have to call the method on line 4 I don't think.